backend/tests/test_auth.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+from backend.database import User
+from backend.auth import user_manager
+from flask_jwt_extended import decode_token
+
```

Comment:
WDYT about a test case for the /test endpoint? That would illustrate how to send the access token in an API request.

Reply:
Added!